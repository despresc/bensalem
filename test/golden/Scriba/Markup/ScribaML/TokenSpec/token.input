{ testcases }

[ various       =,escapes { \#\&\\\]\[\{\} } ]

#leveltag and insignificant #

##anotherleveltag

&layouttag
  [content]

    &subtag
      {content}
      #leveltagmore

      ##leveltag4


      #leveltag5
   &othersub

###thirdtag

#extratag

\inlineelt
