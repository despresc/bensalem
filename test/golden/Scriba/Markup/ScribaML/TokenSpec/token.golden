(0,1,1)(1,1,2) StartBraceGroup
(1,1,2)(2,1,3) LineSpace 1
(2,1,3)(11,1,12) PlainText "testcases"
(11,1,12)(12,1,13) LineSpace 1
(12,1,13)(13,1,14) EndBraceGroup
(13,1,14)(15,3,1) Blanks "\n\n"
(15,3,1)(16,3,2) StartAttrSet
(16,3,2)(17,3,3) LineSpace 1
(17,3,3)(24,3,10) PlainText "various"
(24,3,10)(31,3,17) LineSpace 7
(31,3,17)(32,3,18) Equals
(32,3,18)(33,3,19) Comma
(33,3,19)(40,3,26) PlainText "escapes"
(40,3,26)(41,3,27) LineSpace 1
(41,3,27)(42,3,28) StartBraceGroup
(42,3,28)(43,3,29) LineSpace 1
(43,3,29)(45,3,31) Escape EscNum
(45,3,31)(47,3,33) Escape EscAnd
(47,3,33)(49,3,35) Escape EscBackslash
(49,3,35)(51,3,37) Escape EscRbracket
(51,3,37)(53,3,39) Escape EscLbracket
(53,3,39)(55,3,41) Escape EscLbrace
(55,3,41)(57,3,43) Escape EscRbrace
(57,3,43)(58,3,44) LineSpace 1
(58,3,44)(59,3,45) EndBraceGroup
(59,3,45)(60,3,46) LineSpace 1
(60,3,46)(61,3,47) EndAttrSet
(61,3,47)(63,5,1) Blanks "\n\n"
(63,5,1)(72,5,10) LevelTag 1 "leveltag"
(72,5,10)(73,5,11) LineSpace 1
(73,5,11)(76,5,14) PlainText "and"
(76,5,14)(77,5,15) LineSpace 1
(77,5,15)(90,5,28) PlainText "insignificant"
(90,5,28)(91,5,29) LineSpace 1
(91,5,29)(92,5,30) PlainText "#"
(92,5,30)(94,7,1) Blanks "\n\n"
(94,7,1)(111,7,18) LevelTag 2 "anotherleveltag"
(111,7,18)(113,9,1) Blanks "\n\n"
(113,9,1)(123,9,11) LayoutTag "layouttag"
(123,9,11)(126,10,3) Blanks "\n  "
(126,10,3)(127,10,4) StartAttrSet
(127,10,4)(134,10,11) PlainText "content"
(134,10,11)(135,10,12) EndAttrSet
(135,10,12)(141,12,5) Blanks "\n\n    "
(141,12,5)(148,12,12) LayoutTag "subtag"
(148,12,12)(155,13,7) Blanks "\n      "
(155,13,7)(156,13,8) StartBraceGroup
(156,13,8)(163,13,15) PlainText "content"
(163,13,15)(164,13,16) EndBraceGroup
(164,13,16)(171,14,7) Blanks "\n      "
(171,14,7)(184,14,20) LevelTag 1 "leveltagmore"
(184,14,20)(192,16,7) Blanks "\n\n      "
(192,16,7)(203,16,18) LevelTag 2 "leveltag4"
(203,16,18)(203,16,18) EndImplicitScope
(203,16,18)(203,16,18) EndImplicitScope
(203,16,18)(212,19,7) Blanks "\n\n\n      "
(212,19,7)(222,19,17) LevelTag 1 "leveltag5"
(222,19,17)(222,19,17) EndImplicitScope
(222,19,17)(222,19,17) EndImplicitScope
(222,19,17)(226,20,4) Blanks "\n   "
(226,20,4)(235,20,13) LayoutTag "othersub"
(235,20,13)(235,20,13) EndImplicitScope
(235,20,13)(235,20,13) EndImplicitScope
(235,20,13)(237,22,1) Blanks "\n\n"
(237,22,1)(248,22,12) LevelTag 3 "thirdtag"
(248,22,12)(248,22,12) EndImplicitScope
(248,22,12)(248,22,12) EndImplicitScope
(248,22,12)(248,22,12) EndImplicitScope
(248,22,12)(250,24,1) Blanks "\n\n"
(250,24,1)(259,24,10) LevelTag 1 "extratag"
(259,24,10)(261,26,1) Blanks "\n\n"
(261,26,1)(271,26,11) InlineTag "inlineelt"
(271,26,11)(272,27,1) Blanks "\n"
(272,27,1)(272,27,1) EndImplicitScope
